app/models/micropost.rb

```diff
@@ -1,2 +1,22 @@
 class Micropost < ApplicationRecord
+  MAX_IMAGE_SIZE = 5.megabytes
+  MICROPOST_PERMIT = %i(content image).freeze
+
+  belongs_to :user
+  has_one_attached :image
+
+  scope :newest, ->{order(created_at: :desc)}
```

Comment:
```suggestion
  scope :newest, -> {order(created_at: :desc)}
```
Setup trong rubucop để thêm space trước block e nha